### PR TITLE
Additional probes, increase slave timeout, patch to master.pm for master-slave setups

### PR DESCRIPTION
**Pull request by deveshmanish**
* Fixed Master.pm to properly handle master-slave setup
    * Increased Slave polling timeout from 60s to 300s to give slaves ample time to poll multiple targets.
    * Added additional probes: 
        * speedtest.pm - Ookla official speedtest client
        * speedtest-cli.pm - Sievel's speedtest-cli
        * YoutubeDL.pm - YoutubeDL probe
        * WifiParam.pm - WiFi parameter probe with iw

### CODE REVIEW

File: lib/Smokeping/Master.pm [hunk 1]
```diff
@@ -123,7 +123,7 @@ sub save_updates {
         }
     }
     for my $name (sort keys %u){
-        my $file = slavedatadir($cfg) ."/${name}.${slave}.slave_cache";
+        my $file = slavedatadir($cfg) ."${name}.${slave}.slave_cache";
```
Comment (oetiker) [inline lib/Smokeping/Master.pm:126]: slavedatadir never ends in a slash ... so dropping the slash here has drastic consequences ...

File: release-smokeping.sh [hunk 2]
```diff
@@ -1,9 +1,7 @@
 #!/bin/sh
```
Comment (oetiker) [inline release-smokeping.sh:1]: changing my release process is not ideal :) could you create a separate PR for that

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,51 @@
+FROM debian
```
Comment (oetiker) [inline Dockerfile:1]: having a dockerfile is nice, but it would require some documentation as to how it is intended to be used ...

**Comment by oetiker** [comment]
thank you for your contributions ... love the new probes ...

File: release-smokeping.sh (hunk 2 above)
Comment (deveshmanish) [reply release-smokeping.sh:1]: Seems I missed replacing your version before opening the pull request my bad.

**Comment by deveshmanish** [comment]
Opened a separate PR

**Pull request closed without merging**